include/swift/Option/FrontendOptions.td

```diff
@@ -30,6 +30,9 @@ def primary_filelist : Separate<["-"], "primary-filelist">,
   HelpText<"Specify primary inputs in a file rather than on the command line">;
 def output_filelist : Separate<["-"], "output-filelist">,
   HelpText<"Specify outputs in a file rather than on the command line">;
+def supplementary_output_filemap : Separate<["-"], "supplementary-output-filemap">,
```

Comment:
Nitpick: The term is "output file map", while `-output-filelist` really is "filelist" (for the silly reason that the equivalent linker flag is spelled `-filelist` rather than `-file_list`, when the Apple linker has plenty of other underscore-separated flags). So please use `file-map` here.

Additionally, is it worth putting "supplementary" in the command-line flag? I know we haven't decided yet whether it makes sense to pull primary output files from here too, and it definitely makes sense to use "supplementary" in all the frontend function names. And it doesn't *really* matter because it's a frontend flag. But still, what do you think about just using `-output-file-map`?

Reply:
Changed `filemap` to `file-map` or `file_map` as needed.

I like the idea of just using `-output-file-map`, but prefer to do it as part of a larger merge, unifying primary and supplementary outputs. Even if not, I would defer it to a next PR.